src/command_modules/azure-cli-vm/tests/test_vm_commands.py

```diff
@@ -1656,10 +1656,10 @@ def set_up(self):
     def test_vmss_vms(self):
         self.execute()
 
-    def _check_vms_power_state(self, expected_power_state):
+    def _check_vms_power_state(self, expected_power_states):
```

Comment:
The following definition can make the usage easier:
```
def _check_vms_power_state(self, *args):
```
The client will look like:
```
self._check_vms_power_state('PowerState/running', 'PowerState/starting')
```
This save the extra step to create a list.

Besides, I don't think the method has to initialize with `_`. It is meaningless given this is merely a test case.